treeherder/model/sql/jobs.json

```diff
@@ -18,6 +18,7 @@
                 `signature`,
                 `job_coalesced_to_guid`,
                 `result_set_id`,
+                `push_id`,
```

Comment:
It looks like it does to me? push_id is added to the jobs table in the very first commit.